values.yaml

```diff
@@ -336,4 +336,24 @@ container_images:
         repository: oauth2-proxy/oauth2-proxy
         tag: v7.9.0@sha256:37c1570c0427e02fc7c947ef2c04e8995b8347b7abc9fcf1dbb4e376a4b221a7
   
-enable_chisel_proxy_protocol: false
\ No newline at end of file
+enable_chisel_proxy_protocol: false
+
+node_ports:
```

Comment:
Can you move this node_ports to the very top so it's near the host_ports definition? This way it's all grouped close by.